src/Symfony/Component/HttpFoundation/Tests/ParameterBagTest.php

```diff
@@ -378,6 +378,30 @@ public function testGetEnumThrowsExceptionWithInvalidValueType()
 
         $this->assertNull($bag->getEnum('invalid-value', FooEnum::class));
     }
+
+    public function testGetStringOrNull(): void
+    {
+        $bag = new ParameterBag(['valid-value' => 'foo']);
+
+        $this->assertSame('foo', $bag->getStringOrNull('valid-value'));
+        $this->assertNull($bag->getStringOrNull('invalid-key'));
+    }
+
+    public function testGetIntOrNull(): void
+    {
+        $bag = new ParameterBag(['valid-value' => 1]);
+
+        $this->assertSame(1, $bag->getIntOrNull('valid-value'));
+        $this->assertNull($bag->getIntOrNull('invalid-key'));
```

Comment:
I would name it `missing-key` for clarity.

Reply:
Renamed